packages/nautilus.rb

```diff
@@ -53,6 +52,7 @@ class Nautilus < Package
   depends_on 'pango' # R
   depends_on 'tracker3_miners'
   depends_on 'tracker3' # R
+  depends_on 'vulkan_headers' # R
```

Comment:
This is a runtime dependency? (Isn't this a build dependency?)

Reply:
I'm not sure.  I just noticed an error when compiling.